CDCgov/prime-reportstream
# Add initial overview of universal pipeline

This PR adds a high level overview of the Universal Pipeline. It gets a bit technical but tries to remain accessible for non-technical readers as well. Where appropriate, it links to other more technical documents to provide more information for those interested (likely technical folks working on the pipeline). The goal of this document is to help anyone understand what the Universal Pipeline is, why it exists, and how it works at a high level.

Test Steps:
- Open `prime-router/docs/universal-pipeline-overview/README.md` and ensure all links work
- Ensure language is clear and concise
- Ensure the document meets its stated goals

## Changes
- Updated `prime-router/docs/universal-pipeline-overview/README.md` to contain the main overview information
- Created empty files that should be completed at some point to support the links in README.md
- Added png assets to link to README.md

## Linked Issues
- #10217 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- The following issues should be completed to ensure the documents linked in README.md are completed:
- Complete prime-router/docs/design/metadata.md - #10629
- Complete prime-router/docs/design/swagger.md - #10224
